internal/page/fixtures/attorney.go

```diff
@@ -64,7 +65,7 @@ func Attorney(
 		)
 
 		if attorneySub == "" {
-			attorneySub = random.String(16)
```

Comment:
I really doubt that 62^16 isn't enough, so would be tempted to say put these back so we can see the actual change here